ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -92,6 +92,23 @@
                             </nhs-date-input>
                         </nhs-form-group>
                     </nhs-grid-column>
+                    <nhs-grid-column grid-column-width="OneQuarter">
+                        @{
+                            var PostcodeError = !Model.validationService.IsValid("SearchParameters.Postcode");
+                            var PostcodeGroupState = PostcodeError ? Error : Standard;
+                        }
+                        <nhs-form-group nhs-form-group-type=@PostcodeGroupState>
+                            <label nhs-label-type="Standard"> Postcode </label>
+                            <nhs-hint nhs-hint-type="Standard" id="two-character-postcode-hint">
+                                Enter at least two characters
+                            </nhs-hint>
+                            <span nhs-span-type="ErrorMessage" id="postcode-error"

```

Comment:
Turns out there's a problem with adding these everywhere - they are still visible to screen readers.
@conorsheehansw has created an [really neat fix for it](https://github.com/PublicHealthEngland/ntbs_Beta/blob/master/ntbs-service/TagHelpers/ValidationSpanVisibleTagHelper.cs) - can you apply it here and in other error spans on the page?
We also have a [tech debt](https://airelogic-nis.atlassian.net/browse/NTBS-496) ticket for fixing this on pages retrospectively,